### ISSUE TITLE
Add MuPDFWinRT.dll to Windows phone 8 App 

### ISSUE THREAD

**Issue by eng-dev** [MishaUliutin/MuPDF.WinRT#22]
I can't add this reference to my wp8 application

**Comment by MishaUliutin** [comment]
What do you mean? Could share your exception?

BR,
Misha U.

**Comment by eng-dev** [comment]
i only want to display pdf file in my wp8 application so i add FileViewerPageViewModel.cs &PdfImageViewerDataSource.cs when i add MuPDFWinRT.dll i use vs2012  i found this 

![untitled](https://cloud.githubusercontent.com/assets/2079562/3255259/d2f7beb0-f207-11e3-925c-06b8c37dda82.jpg)

**Comment by MishaUliutin** [comment]
hmm, could you check please that you use right version of MuPDF for right platform? (ARM for ARM, x86 for x86)?
You could find example there - https://bitbucket.org/mishauliutin/pdf-reader-for-wp8-public

**Comment by eng-dev** [comment]
I uploaded  my project Could you please  look at it and give me your feedbackhttps://onedrive.live.com/redir?resid=7BAC31E811886739!341&authkey=!AMPXLwLjX9icOzY&ithint=file%2c.rar

**Comment by eng-dev** [comment]
The problem in .Net Framework version Could you please upgrade it from .NET Framework 3.5 to .NET Framework 4.5?

**Comment by MishaUliutin** [comment]
no, you could not use .Net Framework 3.5 on WP8 at all, so project is .Net Framework 4.5.
The problem in your project that you should use separate version for x86 and ARM configuration (x86 uses by Emulator, ARM is on device), so you should build (or get from this project https://bitbucket.org/mishauliutin/pdf-reader-for-wp8-public) separate version of dll for x86 and for ARM and than add such code to you csproj file:

```
<Reference Include="MuPDFWinRT, Version=255.255.255.255, Culture=neutral, processorArchitecture=MSIL" Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
  <SpecificVersion>False</SpecificVersion>
  <HintPath>..\xternals\MuPDF\x86\MuPDFWinRT.winmd</HintPath>
</Reference>
<Reference Include="MuPDFWinRT, Version=255.255.255.255, Culture=neutral, processorArchitecture=MSIL" Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
  <SpecificVersion>False</SpecificVersion>
  <HintPath>..\xternals\MuPDF\x86\MuPDFWinRT.winmd</HintPath>
</Reference>
<Reference Include="MuPDFWinRT, Version=255.255.255.255, Culture=neutral, processorArchitecture=MSIL" Condition=" '$(Configuration)|$(Platform)' == 'Debug|ARM' ">
  <SpecificVersion>False</SpecificVersion>
  <HintPath>..\xternals\MuPDF\ARM\MuPDFWinRT.winmd</HintPath>
</Reference>
<Reference Include="MuPDFWinRT, Version=255.255.255.255, Culture=neutral, processorArchitecture=MSIL" Condition=" '$(Configuration)|$(Platform)' == 'Release|ARM' ">
  <SpecificVersion>False</SpecificVersion>
  <HintPath>..\xternals\MuPDF\ARM\MuPDFWinRT.winmd</HintPath>
</Reference>
```

where ..\xternals\MuPDF\ARM - path to ARM version of MuPDF dll and ..\xternals\MuPDF\x86 - path to x86 version of dll. You could find example there https://bitbucket.org/mishauliutin/pdf-reader-for-wp8-public/src/8974b17561239823f1974ece7458ad35979f4532/PdfReader/PdfReader.csproj?at=default

**Comment by eng-dev** [comment]
I did all of that but also can't add MuPDFWinRT.dll
